src/pages-and-resources/discussions/app-config-form/apps/shared/AnonymousPostingFields.jsx

```diff
@@ -14,7 +14,7 @@ function AnonymousPostingFields({
 }) {
   return (
     <>
-      <h5>{intl.formatMessage(messages.anonymousPosting)}</h5>
+      <h4>{intl.formatMessage(messages.anonymousPosting)}</h4>
```

Comment:
I think it should not be changed because as per figma it is supposed to be h5 

<img width="979" alt="Screenshot 2021-04-21 at 10 28 09 PM" src="https://user-images.githubusercontent.com/26253150/115595967-01720600-a2f1-11eb-8268-4d7873ebd15f.png">

Reply:
yes, you are right. I confused this heading with the form label group. I reverted that change